linkcheck/configuration/confparse.py

```diff
@@ -227,6 +227,8 @@ def read_filtering_config (self):
             for line in read_multiline(self.get(section, "nofollow")):
                 pat = get_link_pat(line, strict=0)
                 self.config["externlinks"].append(pat)
+		if self.has_option(section, "ignoreclass"):
+                self.config["ignoreclass"] = [f.strip() for f in self.get(section, 'ignoreclass').split(',')]
```

Comment:
The indentation looks very wrong here.

Are you using literal tab characters instead of spaces for indentation, or something?  Please don't.

Reply:
Okay, the new GitHub notifications UI is messed up, and apparently showed me an outdated diff.

(Good job GitHub!  *slow clap*)

Anyway, the indentation is not entirely correct here either: the code inside the `if` should be indented with 4 spaces, not 8 spaces.

Reply:
Think fixed in later commit.